README.md

```diff
@@ -24,17 +24,55 @@ See the [wiki](https://github.com/NASA-AMMOS/common-workflow-service/wiki) for m
     - You will need to add your own Tomcat keystore file to the `install/` direcotory
     - You will need to add your own truststor file to the `install/tomcat_lib/` directory
     - See: https://tomcat.apache.org/tomcat-9.0-doc/ssl-howto.html
+    
+  - **Apache Maven 3.8.1+**: Install Maven by following these steps:
+    - See: https://maven.apache.org/install.html
+    
+#### Development Environment Configuration
+
+##### Tomcat Server
+
+Start CWS Server:
+* `cd` into `install` folder and run command: 
+
+```
+./start_cws.sh
```

Comment:
@jamesfwood  changed tomcat server command from `./catalina.sh` to `./start_cws.sh`